src/book/mod.rs

```diff
@@ -270,8 +271,13 @@ impl MDBook {
                     _ => continue,
                 };
 
+                if !chapter.is_empty() && ch.name != chapter {
```

Comment:
Can this be an `Option<&str>` instead of using an empty string as a sentinel?

Reply:
Done.